scripts/build/build-utils.ps1

```diff
@@ -85,25 +85,6 @@ function New-NuGetPackages([string]$binPath) {
     }
 }
 
-function Restore-Packages (
```

Comment:
before this commit, I had some weird local failures when running the `dev-build` script with `-restore`, errors like the one below
```
C:\Program Files\dotnet\sdk\5.0.100-rc.2.20479.15\Sdks\Microsoft.NET.Sdk\targets\Microsoft.PackageDependencyResolution.targets(241,5): error NETSDK1005: Assets file 'C:\Workspace\sonar-dotnet\sonaranalyzer-dotnet\tests\SonarAnalyzer.UnitTest\obj\project.assets.json' does n't have a target for 'net48'. Ensure that restore has run and that you have included 'net48' in the TargetFrameworks for your project. [C:\Workspace\sonar-dotnet\sonaranalyzer-dotnet\tests\SonarAnalyzer.UnitTest\SonarAnalyzer.UnitTest.csproj]
```